.github/workflows/ci.yml

```diff
@@ -26,14 +26,12 @@ jobs:
           # https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
           - 1.15.11
           - 1.16.8
-        build_tags: ["", "saas"]
```

Comment:
I would just remove "saas" from here and that would be the best way to include it back if required.

Reply:
ok